src/Model/AbstractModel.php

```diff
@@ -275,10 +275,14 @@ public function getPackagedName($namespaced = false)
         if ($namespaced && $this->getNamespace() !== '') {
             $nameParts[] = sprintf('\%s\\', $this->getNamespace());
         }
+        $cleanName = $this->getCleanName();
         if ($this->getGenerator()->getOptionPrefix() !== '') {
             $nameParts[] = $this->getGenerator()->getOptionPrefix();
+        } else {
+            $cleanName = self::replaceReservedPhpKeyword($cleanName, null);
```

Comment:
As you defined `$context` as `null` by default at line [366](https://github.com/WsdlToPhp/PackageGenerator/pull/46/files#diff-96f379c0a0aae9c0c450923d7d3b98b3R366), why do you pass `null`as second parameter here ?